PlayerCore/components/New VRM Core/VRMItemResponseTime.swift

```diff
@@ -0,0 +1,78 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+
+public struct VRMItemResponseTime {
+    typealias Container = [VRMCore.Item: TimeRange]
+    
+    static let initial = VRMItemResponseTime(timeRangeContainer: [:])
+    
+    public struct TimeRange: Hashable {
+        public let startAt: Date
+        public var finishAt: Date?
+    }
+    
+    public let timeRangeContainer: [VRMCore.Item: TimeRange]
+}
+
+func reduce(state: VRMItemResponseTime, action: Action) -> VRMItemResponseTime {
+    
+    func updateFinishTime(for item: VRMCore.Item,
+                          with date: Date,
+                          in container: VRMItemResponseTime.Container) -> VRMItemResponseTime.Container {
+        guard var range = container[item] else {
+            fatalError("get finish error action for item which wasn't started")
+        }
+        range.finishAt = date
+        var newTimeContainer = container
+        newTimeContainer[item] = range
+        
```

Comment:
Do we need this empty line before return?

Reply:
Removed